.github/workflows/ci.yml

```diff
@@ -5,26 +5,21 @@ on:
   pull_request:
     branches: [ master ]
 
-jobs:
-  build:
+env:
+  AREA: liechtenstein
```

Comment:
If you really want to promote tilemaker usage through GH Actions I would specify a complete (and overridable) download URL since right now you would be limited to European countries and you could not use regional extracts or data sources other than Geofabrik.

Reply:
I thinks this doesn't matter. What `ci.yml` does is just test the current build for `tilemaker`. 
How user apply GH Action is coded inside `action.yml`

Reply:
Ah I see, the download is the responsibility of the "using" action. Then this is a moot point. Ignore me.:)